ext/node/ops/os.rs

```diff
@@ -0,0 +1,196 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::error::AnyError;
+use deno_core::op;
+use deno_core::OpState;
+use errno::errno;
+use errno::set_errno;
+use errno::Errno;
+
+#[op]
+pub fn op_node_os_get_priority<P>(
+  state: &mut OpState,
+  pid: u32,
+) -> Result<i32, AnyError>
+where
+  P: NodePermissions + 'static,
+{
+  {
+    let permissions = state.borrow_mut::<P>();
+    permissions.check_sys("getPriority", "node:os.getPriority()")?;
+  }
+
+  get_priority(pid)
+}
+
+#[op]
+pub fn op_node_os_set_priority<P>(
+  state: &mut OpState,
+  pid: u32,
+  priority: i32,
+) -> Result<(), AnyError>
+where
+  P: NodePermissions + 'static,
+{
+  {
+    let permissions = state.borrow_mut::<P>();
+    permissions.check_sys("setPriority", "node:os.setPriority()")?;
+  }
+
+  set_priority(pid, priority)
+}
+
+#[op]
+pub fn op_node_os_username<P>(state: &mut OpState) -> Result<String, AnyError>
+where
+  P: NodePermissions + 'static,
+{
+  {
+    let permissions = state.borrow_mut::<P>();
+    permissions.check_sys("userInfo", "node:os.userInfo()")?;
+  }
+
+  Ok(whoami::username())
+}
+
+use crate::NodePermissions;
+#[cfg(unix)]
```

Comment:
There are a lot of `#[cfg(...)]` in this module. You may want to consider something like:

```
#[cfg(unix)]
pub use unix::*;

#[cfg(unix)]
mod unix {
  use libc::id_t;
  use libc::PRIO_PROCESS;
  
  pub fn get_priority(pid: u32) -> Result<i32, AnyError> {
    // ...etc...
  }
  
  // ...etc...
}